pom.xml

```diff
@@ -183,6 +183,10 @@
         <dep.objenesis.version>1.3</dep.objenesis.version>
         <dep.slice.version>0.10</dep.slice.version>
         <dep.jmh.version>1.15</dep.jmh.version>
+        <dep.junit.jupiter.version>5.0.0-RC2</dep.junit.jupiter.version>
+        <dep.junit.jupiter.platform.version>1.0.0-RC2</dep.junit.jupiter.platform.version>
+        <dep.junit.version>4.12</dep.junit.version>
+        <dep.junit.vintage.version>${dep.junit.version}.0-RC2</dep.junit.vintage.version>
```

Comment:
This is also unused. I'm not sure how it's intended to be used, but I'd copy the version rather than appending what appears to be a very specific suffix that will likely change in the future (when it's no longer RC).